index.html

```diff
@@ -4,26 +4,137 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="style.css">
     <title>Conference</title>
 </head>
 <body>
+   <section class="block">
     <section class="menu">
-        <div class="social-icons"></div>
-        <nav>
+        <div class="social-icons">
+            <ul class="social-icons-items">
+                <li><img src="images/facebook.png" alt="facebook-icon"/></li>
+                <li><img src="images/twitter.png" alt="twitter-icon"/></li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">My Page</a></li>
+                <li><a href="#">Logout</a></li>
+            </ul>
+        </div>
+        <nav class="nav-items">
             <div class="humburger">
                 <span class="bar"></span>
                 <span class="bar"></span>
                 <span class="bar"></span>
             </div>
-            <div class="logo"></div>
-            <div class="menu-item"></div>
+            <div class="logo">
+                <a href="index.html"><img src="images/logo.png" alt="logo"/></a>
+            </div>
+            <div class="main-menu">
+                <ul class="manu-item">
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="#"> Program</a></li>
+                    <li><a href="#">Join</a></li>
+                    <li><a href="#">Sponsor</a></li>
+                    <li><a href="#"> News</a></li>
+                    <li class="cc-top"><a href="#" class="cc">CC Campaign</a></li>
+                </ul>
+            </div>
         </nav>
     </section>
-    <section class="headline"></section>
-    <section class="main-program"></section>
-    <section class="featured-speakers"></section>
-    <section class="partners"></section>
-    <section class="footer"></section>
+    <section class="headline">
+        <header class="hero-container">
+            <h4>"Hello! Sharing World"</h4>
+            <h1>CREATIVE COMMONS GLOBAL SUMMIT 2015</h1>
+        </header>
+        <div class="hero-content">
+            <p>
+                A joyful celebration believing in the value of openness and sharing, creating a 
+                positive change with people from over 80 countries is taking place in October, 
+                in Korea.
+            </p>
+        </div>
+        <div class="hero-date">
+            <h1>2015.10.15(THU) ~ 16(FRI)</h1>
+            <h4>@ National Museum of Karea, Art Center Nabi and more</h4>
+        </div>
```

Comment:
- [ ] Kindly note that in the project requirement, you are required to personalize the content of your page. Choose a topic that is different than the one in the original design. Your content is exactly the same as the one in the original design.

![image](https://github.com/ctokoli/Conference-Capstone-Project/assets/98453979/a24476bf-e395-4a13-84ff-b6880ee0df06)